gridt/src/app/movements/add-movement/add-movement.page.ts

```diff
@@ -17,6 +17,27 @@ export class AddMovementPage implements OnInit, OnDestroy {
     'twice daily',
     'weekly'
   ];
+  locations: string[] = [
+    'Groningen',
+    'Friesland',
+    'Drenthe',
+    'Overijssel',
+    'Flevoland',
+    'Noord-Holland',
+    'Zuid-Holland',
+    'Gelderland',
+    'Utrecht',
+    'Limburg',
+    'Noord-Brabant',
+    'Zeeland'
+  ];
+  categories: string[] = [
+    'Health & Fitness',
+    'Mind & Learning',
+    'Community',
+    "Sustainable Planet",
+    "Miscellaneous"
+  ];
```

Comment:
Good fix for now. I think it is good if everyone is aware of this such that we can move these lists to the server one day.